src/router/provider.tsx

```diff
@@ -1,5 +1,5 @@
 import { RouterProvider } from "react-router-dom";
-import { router } from "./router";
+import { router } from "./router.tsx";
```

Comment:
I think we do not need .tsx extension here for imports to look cleaner

Reply:
fixed